lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -60,8 +60,8 @@
         description:
             - The Docker networking mode to use for the containers in the task.
             - C(awsvpc) mode was added in Ansible 2.5
-            - Windows containers must use I(network_mode=default), which will utilize docker NAT networking.
-            - Setting I(network_mode=default) for a Linux container will use bridge mode.
+            - Windows containers must use network_mode=default, which will utilize docker NAT networking.
```

Comment:
```suggestion
            - Windows containers must use I(network_mode=default), which will utilize docker NAT networking.
```

Using I(...) adds some extra highlighting that says "Option network_mode set to value 'default'"